frontend/src/static/css/style.scss

```diff
@@ -425,6 +425,7 @@ header{
                     .hljs{
                         // overwrite hljs library
                         display:inherit;
+                        padding:0;
```

Comment:
why is this change necessary?

Reply:
It was because `v-hljs` was moved within `segment`, and the `hljs` library has an in-built padding value which causes paddings in between each `segment` as such:
<img width="460" alt="Screen Shot 2019-06-18 at 11 42 35 PM" src="https://user-images.githubusercontent.com/32864116/59698921-3b6caf80-9223-11e9-8380-ed126b2a7429.png">

So the reason for overwriting the padding is to eliminate the spacing between each segment.